### DISCUSSION TITLE
analyze_from_initial_mesh()

### DISCUSSION THREAD

**Discussion by wdmaiansys** [ansys/pyaedt#1670] [Q&A] [answered]
I did not see any input parameter for analyze_from_initial_mesh()

_Hfss.analyze_from_initial_mesh()
Revert the solution to the initial mesh and re-run the solve.

Returns:
[bool](https://docs.python.org/dev/library/stdtypes.html#bltin-boolean-values)
True when successful, False when failed._

How could I send it parameter to do that on a specific setup?

**Comment by Samuelopez-ansys** [comment, marked as answer]
Hi @wdmaiansys ,

Please for future questions, go to the Ansys developer portal: https://discuss.ansys.com/ 

If you find a bug or a new feature, please submit an Issue in Github. 

About your question, the setup is specified with this property:

hfss.analysis_setup

You could change the 'nominal' setup like:
app.analysis_setup = "MyNewSetup"

And then if you use this operation "analyze_from_initial_mesh()" will apply to "MyNewSetup". I agree this could improve, please submit an issue.
